redash/models.py

```diff
@@ -830,6 +830,10 @@ def recent(cls, group_ids, user_id=None, limit=20):
 
         return query
 
+    @classmethod
+    def get_by_id(cls, _id):
+        return db.session.query(cls).filter(cls.id == _id).one()
```

Comment:
@arikfr Maybe `cls.query[_id]` is better? `.get()` returns `None` and that's why it shows wrong exception message.